src/backend/src/main/java/com/linkedpipes/lpa/backend/controllers/VirtuosoController.java

```diff
@@ -21,4 +32,13 @@
     public ResponseEntity<String> serviceDescription(@RequestParam(value = "graphId") String graphId) {
         return ResponseEntity.ok(TtlGenerator.getVirtuosoServiceDescription(GRAPH_NAME_PREFIX + graphId));
     }
+
+    @GetMapping("api/virtuoso/graphExists")
+    public ResponseEntity checkNamedGraphExists(@RequestParam(value = "graphId") String graphId) throws LpAppsException {
+        boolean exists = virtuosoService.checkNamedGraphExists(graphId);
```

Comment:
Shouldn't it be `GRAPH_NAME_PREFIX + graphId` here?

Reply:
Doesn't apply to this case